.env.example

```
@@ -29,8 +35,36 @@ DASHBOARD_PASSWORD=this_password_is_insecure_and_should_be_updated
+# [required]
+# Supabase Storage
+# These are required by the latest Supabase storage-api container.
+# The default "stub" values work fine when using file-based storage (not S3).
+# For S3 protocol access keys, generate secure values for production.
+# Docs: https://supabase.com/docs/guides/self-hosting/self-hosted-s3
+############
+
+GLOBAL_S3_BUCKET=stub
+REGION=stub
+STORAGE_TENANT_ID=stub
+S3_PROTOCOL_ACCESS_KEY_ID=625729a08b95bf1b7ff351a663f3a23c
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_PROTOCOL_ACCESS_KEY_ID key should go before the STORAGE_TENANT_ID key

(UnorderedKey)